src/form/Select/Select.tsx

```diff
@@ -638,6 +637,45 @@ export const Select: FC<Partial<SelectProps>> = ({
     [createable, menuDisabled, onInputBlur, toggleSelectedOption]
   );
 
+  const onPasteHandler = useCallback(
+    (e: React.ClipboardEvent<HTMLInputElement>) => {
+      if (createable && multiple) {
```

Comment:
This needs to be an option `createOnPaste` or `selectOnPaste` might actually be more correct.